rexmex/scorecard.py

```diff
@@ -91,3 +92,68 @@ def print_metrics(self):
         Printing the name of metrics.
         """
         print({k for k in self.metric_set.keys()})
+
+
+class CoverageScoreCard(ScoreCard):
+    """
+    Coverage scorecard can be used to aggregate coverage-related metrics, plot those, and generate performance reports.
+    """
+
+    def __init__(self, metric_set: Mapping[str, Metric], all_users: Collection[str], all_items: Collection[str]):
+        """
+        all_users and all_items define the relevant user and item space.
+        """
+
+        super().__init__(metric_set)
+        self.all_users = all_users
+        self.all_items = all_items
+
+    def get_coverage_metrics(self, recommendations: List[Tuple]) -> pd.DataFrame:
```

Comment:
This should override `get_performance_metrics` instead of making a new function

Reply:
Since this really changes the interface of a score card, perhaps there is a need for an alternate, higher-level abstraction for a score card that only has the `generate_report()` function

Reply:
I also had considered that the score card class could be merged with the metric set classes, too